eip/EIPS/eip-4337.md

```diff
@@ -30,6 +30,17 @@ This proposal takes a different approach, avoiding any adjustments to the consen
     * Privacy-preserving applications
     * Atomic multi-operations (similar goal to EIP-3074)
     * Pay tx fees with ERC-20 tokens, allow developers to pay fees for their users, and [EIP-3074](./eip-3074.md)-like **sponsored transaction** use cases more generally
+    * Support aggregated signature (e.g. BLS)
+
+## Definitions
+
+* **UserOperation** - a structure that describes a transaction to be sent on behalf of a user. To avoid confusion, it is named "transaction".
+  * Like a transaction, it contains "sender", "to", "calldata", "maxFeePerGas", "maxPriorityFee", "signature", "nonce"
+  * unlike transaction, it contains several other fields, desribed below
+  * also, the "nonce" and "signature" fields usage is not defined by the protocol, but by each account implementation
+* **Sender** - the account contract sending a user operation.
+* **EntryPoint** - a singleton contract to execute bundles of UserOperations. Bundlers/Clients whitelist the supported entrypoint.
+* **Aggregator** - a helper contract trusted by wallets to validate an aggregated signature. Bundlers/Clients whitelist the supported aggregators.
```

Comment:
Validator validates. Verifier verifies. Helper helps. And aggregator aggregates. 

Here, `aggregator`, however, helps. And validates. But does not aggregate. 

I suggest we try finding for a better name? 😀

Reply:
Yes, we do need to think about naming.
My problem is that the "Aggregator" is the off-chain module that creates the aggregated sig.
This component is "de-aggregator", or "aggregatedSignatureValidator" - but I don't like such long names...